source/core/game/Game.gd

```diff
@@ -14,7 +14,8 @@ func _unhandled_input(event: InputEvent) -> void:
 		var location: Location = scenario.map.get_location(mouse_cell)
 		if location.movable:
 			selected_unit = location.movable
-			selected_unit.reachable = scenario.map.find_all_reachable_cells(selected_unit)
+			selected_unit.reachable =\
+					scenario.map.find_all_reachable_cells(selected_unit)
```

Comment:
No need to split this onto a new line. We're not enforcing a strict max line length.